px-checkout/src/main/java/com/mercadopago/android/px/internal/features/explode/ExplodingFragment.kt

```diff
@@ -34,7 +34,9 @@ import com.mercadopago.android.px.internal.extensions.visible
 import com.mercadopago.android.px.internal.util.ViewUtils
 import kotlin.math.hypot
 
-class ExplodingFragment : Fragment() {
+private const val MAX_LOADING_TIME = 20000
```

Comment:
We could add a time unit in the const name and also use the underscore as thousand separator for better readability

```suggestion
private const val MAX_LOADING_TIME_MILLIS = 20_000
```